pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -1550,3 +1550,4 @@ https://github.com/ziglang/zig.vim/,,
 https://github.com/zk-org/zk-nvim/,HEAD,
 https://github.com/troydm/zoomwintab.vim/,,
 https://github.com/nanotee/zoxide.vim/,,
+https://github.com/tummetott/unimpaired.nvim/,HEAD,
```

Comment:
Can you move this to its sorted position, please.

Reply:
Done